consensus/cosipbft/queue.go

```diff
@@ -148,3 +149,10 @@ func (q *queue) Finalize(to Digest, sig crypto.Signature) (*ForwardLinkProto, er
 
 	return packed.(*ForwardLinkProto), nil
 }
+
+func (q *queue) Clear() {
+	q.Lock()
+	q.locked = false
+	q.items = nil
```

Comment:
isn't it dangerous to set it to nil? I would expect a clear to empty the list, not "invalidate" it.

Reply:
https://play.golang.org/p/WYEvj6L0rkD

Reply:
learned something today